test/testHelper.ts

```diff
@@ -26,7 +26,10 @@ export default class ContextTestingHelper {
   }
 
   createDummyContextFile(): void {
-    writeFileSync(DEFAULT_CONTEXT_FILE_PATH, JSON.stringify(this._context), { encoding: 'utf-8' });
+    writeFileSync(DEFAULT_CONTEXT_FILE_PATH, JSON.stringify(this._context), { encoding: 'utf-8', flag: 'w+' });
+    if (!existsSync(DEFAULT_CONTEXT_FILE_PATH)) {
+      throw new Error('Context file doesn\'t exist');
+    }
```

Comment:
@derberg I added this check, and it seems like the file exists. The action doesn't throw this error. Let me know if I should add the check anywhere else.